packages/flutter/lib/src/material/snack_bar.dart

```diff
@@ -9,10 +9,10 @@ import 'button_theme.dart';
 import 'flat_button.dart';
 import 'material.dart';
 import 'scaffold.dart';
+import 'snack_bar_theme.dart';
 import 'theme.dart';
 import 'theme_data.dart';
 
-const double _kSnackBarPadding = 24.0;
 const double _kSingleLineVerticalPadding = 14.0;
 const Color _kSnackBackground = Color(0xFF323232);
```

Comment:
Perhaps in another PR but I think it would be better as

```suggestion
const Color _kSnackBackgroundColor = Color(0xFF323232);
```

Reply:
Done